packages/next/types/index.d.ts

```diff
@@ -19,14 +19,23 @@ declare module 'react' {
 
   // <style jsx> and <style jsx global> support for styled-jsx
   interface StyleHTMLAttributes<T> extends HTMLAttributes<T> {
-    jsx?: boolean;
-    global?: boolean;
+    jsx?: boolean
+    global?: boolean
   }
 }
 
+
+/**
+ * `Page` component, used for rendering `pages`.
```

Comment:
In this case `Page` is only a Typescript type
```suggestion
 * `Page` type, use it as a guide to create `pages`.
```